src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -253,12 +253,24 @@ def initialize_newborns(self, pop_data: SimulantData) -> None:
             "relation_to_household_head"
         ].map(metadata.NEWBORNS_RELATION_TO_HOUSEHOLD_HEAD_MAP)
 
-        # Make age negative so age + step size gets simulants correct age at the end of the time step (ref line 284)
-        new_births["age"] = -self.randomness.get_draw(new_births.index, "age") * (
-            self.step_size_days / DAYS_PER_YEAR
-        )
-        new_births["date_of_birth"] = pop_data.creation_time - pd.to_timedelta(
-            np.round(new_births["age"] * DAYS_PER_YEAR), unit="days"
+        # birthday map between parent_ids and DOB (so twins get same bday)
+        # note we use np.floor to guarantee birth at midnight
+        dob_map = {
+            parent: dob for (parent, dob) in zip(
+                parent_ids.unique(),
+                pop_data.creation_time + pd.to_timedelta(
+                    np.floor(self.randomness.get_draw(parent_ids.unique(), "dob") * self.step_size_days), unit="days"
+                )
```

Comment:
is parent_ids an Index? I assume it must be since you're passing it into `get_draw`. Maybe it'd be good to label `parent_ids` `parent_ids_index` instead?